utils/results/conversion/sarifparser/sarifparser.go

```diff
@@ -0,0 +1,408 @@
+package sarifparser
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/jfrog/gofrog/datastructures"
+	"github.com/jfrog/jfrog-cli-security/utils"
+	"github.com/jfrog/jfrog-cli-security/utils/formats"
+	"github.com/jfrog/jfrog-cli-security/utils/formats/sarifutils"
+	"github.com/jfrog/jfrog-cli-security/utils/jasutils"
+	"github.com/jfrog/jfrog-cli-security/utils/results"
+	"github.com/jfrog/jfrog-cli-security/utils/severityutils"
+	"github.com/jfrog/jfrog-cli-security/utils/techutils"
+	"github.com/jfrog/jfrog-client-go/xray/services"
+	"github.com/owenrumney/go-sarif/v2/sarif"
+)
+
+const (
+	FixedVersionSarifPropertyKey = "fixedVersion"
+	WatchSarifPropertyKey        = "watch"
+
+	ScaToolName  = "JFrog Xray SCA"
+	SastToolName = "USAF"
+	IacToolName  = "JFrog Terraform scanner"
+	// #nosec G101 -- Not credentials.
+	SecretsToolName           = "JFrog Secrets scanner"
+	ContexualAnalysisToolName = "JFrog Applicability Scanner"
```

Comment:
This const is not in use anywhere, but if you do need it fix typo:
ContexualAnalysisToolName  -> ContextualAnalysisToolName